src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -173,6 +173,9 @@ public static Calendar convertToUserTimeZone(Calendar time, double timeZone) {
      */
     @Deprecated
     public static Date convertLocalDateToUtc(Date localDate, double localTimeZone) {
+        if (localDate == null) {
+            return null;
+        }
```

Comment:
Instead of doing the null check here, I think it would be better suited to do it in `getStartTimeString` and `getEndTimeString` so that it is clearer that the behaviour is different when the value is null.